src/btf.h

```diff
@@ -72,6 +79,12 @@ class BTF
 
   std::pair<int, int> get_btf_id_fd(const std::string& func,
                                     const std::string& mod) const;
+  // get dump id and type id
+  std::pair<int, int> get_dumpid_typeid(const std::string& type_name);
```

Comment:
The dump id/type id pair is also used in other places outside of this class. Maybe it'd be worth defining a type for it?

Reply:
Ok, will do, thanks!